test/HashContainerTest/HashMap.test.ts

```diff
@@ -103,6 +102,25 @@ describe('HashMap test', () => {
     judgeHashMap(myHashMap, stdMap);
   });
 
+  test('HashMap eraseElementByKey function test2', () => {
+    const hashMap = new HashMap<string, number>(undefined, undefined, () => 0);
+
+    const mockHashTable: OrderedMap<string, number>[] = [];
+
+    const orderedMap1 = new OrderedMap<string, number>();
+    orderedMap1.setElement('1', 1);
+    orderedMap1.setElement('2', 2);
+    orderedMap1.setElement('3', 3);
+    orderedMap1.setElement('4', 4);
+    orderedMap1.setElement('5', 5);
+    orderedMap1.setElement('6', 6);
+    mockHashTable.push(orderedMap1);
+    // @ts-ignore
+    hashMap.hashTable = mockHashTable;
```

Comment:
This is a dangerous operation because the purpose of testing is to verify correctness not to improve test coverage.

Do we have a better way?